src/librustc_resolve/lib.rs

```diff
@@ -14,6 +14,7 @@
 #![feature(crate_visibility_modifier)]
 #![feature(label_break_value)]
 #![feature(nll)]
+#![feature(slice_patterns)]
```

Comment:
Make sure to change this to `cfg_attr(bootstrap, ...)` if https://github.com/rust-lang/rust/pull/67712 lands first.